Source/astcenc_compress_symbolic.cpp

```diff
@@ -1399,19 +1405,16 @@ float compress_symbolic_block(
 			break;
 		}
 
-		for (int i = 0; i < 2; i++)
+
+		if (lowest_correl <= ctx.config.tune_two_plane_early_out_limit)
```

Comment:
Switched the direction comparison on this - intentional?

Reply:
Yes - it flipped from being a "disable if higher than limit" to being an "enable if lower than the limit".